projects/wallet-template/STEP-BY-STEP-GUIDE.md

```diff
@@ -0,0 +1,134 @@
+# Step by Step Guide
+
+To add light client functionality to your dapp, follow this step by step guide.
```

Comment:
as commented above, it would be nice to add a bit more detail here on what this guide is about.